src/mainwindow.cpp

```diff
@@ -92,6 +93,21 @@ MainWindow::MainWindow(QWidget *parent) :
     ui->graphicsViewChipset->scale(2.0, 2.0);
     ui->graphicsViewMap->scale(2.0, 2.0);
 
+
+    //initialising size of qsplitter of general tab
+    QDesktopWidget *desktop = new QDesktopWidget;
+    int desktopWidth = desktop->screenGeometry().width();
+    int desktopHeight = desktop->screenGeometry().height();
+
+    QList<int> desktopSizeListWidth;
+    desktopSizeListWidth.append(desktopWidth/5);
+    desktopSizeListWidth.append(3*desktopWidth/5);
+    ui->splitter_2->setSizes(desktopSizeListWidth);
```

Comment:
You put a `splitter_2` into the `ui` private member, so I bet you can do the same with `desktop`. ;) (See my previous comment)